tests/ballerina-unit-test/src/test/resources/test-src/lock/locks-in-functions.bal

```diff
@@ -106,6 +106,7 @@ function throwErrorInsideLock() returns (int, string) {
         lockWithinLockString1 = "second worker string";
         lockWithinLockInt1 = lockWithinLockInt1 + 50;
     }
+    error? waitResult = trap wait w1;
```

Comment:
Don't we need to check whether any error has occurred?

Reply:
We have written this to intentionally get an error. But this is a lock test. No point of checking it. We only need to check the locks are getting released properly after a panic within a lock.